app/models/user.rb

```diff
@@ -1,5 +1,20 @@
+require 'uri'
 class User < ApplicationRecord
   # This is a module, part of ActiveModel library
   # full details here: https://api.rubyonrails.org/classes/ActiveModel/SecurePassword/ClassMethods.html
   has_secure_password
+
+  validates_length_of :password, :in => 6..10, :on => :create
+
+  validates_presence_of :first_name
+  validates_presence_of :last_name
+  validates_presence_of :email
+  validates_uniqueness_of :email
+
+  EmailRegex = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
is this naming convention correct? if just ordinary variable -> lower case

Reply:
That's the naming convention